encord/objects/classification_instance.py

```diff
@@ -221,6 +221,9 @@ def set_for_frames(
             last_edited_at = datetime.now()
 
         if self._range_only:
+            # TODO: Enforce checking of ranges (here or on BE)
```

Comment:
are you going to do that in this PR?

Reply:
No, I've decided to do this in the BE actually.

Reply:
Wil remove this comment

Reply:
Actually, will enforce it here as well.